gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/FileImporter.java

```diff
@@ -39,69 +38,72 @@
  */
 @Service
 public class FileImporter {
-    private static final Logger LOGGER = LoggerFactory.getLogger(FileImporter.class);
+    private final UrlWhitelistConfiguration urlWhitelistConfiguration;
+    private final Logger businessLogger;
 
-    private final UrlValidationService urlValidationService;
-
-    public FileImporter(UrlValidationService urlValidationService) {
-        this.urlValidationService = urlValidationService;
+    public FileImporter(UrlWhitelistConfiguration urlWhitelistConfiguration, Logger businessLogger) {
+        this.urlWhitelistConfiguration = urlWhitelistConfiguration;
+        this.businessLogger = businessLogger;
     }
 
     public TcDocumentType importTtcAdjustment(String ttcUrl) {
-        try (InputStream inputStream = urlValidationService.openUrlStream(ttcUrl)) {
+        try {
```

Comment:
try-with-ressources are better unless you close yourself the stream in a "finally" block to avoid ressource to leak.